<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IconCacheSetupDialog</class>
 <widget class="QDialog" name="IconCacheSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Icon cache database setup</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkBoxEnableIconCacheDatabase">
     <property name="toolTip">
      <string>Enable the icon cache database</string>
     </property>
     <property name="text">
      <string>Enable the icon cache database</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelIconCacheDatabase">
       <property name="text">
        <string>Icon cache database</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditIconCacheDatabase">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Icon cache database file (write)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonBrowseIconCacheDatabase">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Browse icon cache database file</string>
       </property>
       <property name="icon">
        <iconset resource="qmc2.qrc">
         <normaloff>:/data/img/fileopen.png</normaloff>:/data/img/fileopen.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="toolTip">
        <string>Apply changes and close dialog</string>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="toolTip">
        <string>Revert changes and close dialog</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="toolButtonImportIcons">
     <property name="toolTip">
      <string>Reimport icons to the cache database now</string>
     </property>
     <property name="text">
      <string>Reimport icons now</string>
     </property>
     <property name="icon">
      <iconset resource="qmc2.qrc">
       <normaloff>:/data/img/rebuild.png</normaloff>:/data/img/rebuild.png</iconset>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBoxEnableIconCacheDatabase</tabstop>
  <tabstop>toolButtonImportIcons</tabstop>
  <tabstop>lineEditIconCacheDatabase</tabstop>
  <tabstop>toolButtonBrowseIconCacheDatabase</tabstop>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources>
  <include location="qmc2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkBoxEnableIconCacheDatabase</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelIconCacheDatabase</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxEnableIconCacheDatabase</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditIconCacheDatabase</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxEnableIconCacheDatabase</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolButtonBrowseIconCacheDatabase</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>IconCacheSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>IconCacheSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
